src/templates/career-details.tsx

```diff
@@ -83,6 +88,13 @@ const CareerPage: React.FC<CareerPageProps> = (props): JSX.Element => {
   const { t } = useTranslation();
 
   const ref = React.useRef<HTMLDivElement | null>(null);
+  const breadcrumb = [
+    ...pageContext.breadcrumb,
+    {
+      pathname: pageContext.i18n.originalPath,
+      label: pageContext.position.name,
+    },
+  ];
```

Comment:
The same goes for this template. I don't see the benefit of having parts of the breadcrumb in the page context. I suggest to remove the `BREADCRUMB_*` bit from the `create-pages` script and instead hardcode them here - including the usage of `t(...)`:
```suggestion
  const breadcrumb = [
    { pathname: '/', label: 'Satellytes' },
    { pathname: '/career', label: t('navigation.career') },
    {
      pathname: pageContext.i18n.originalPath,
      label: pageContext.position.name,
    },
  ];
```